deployment/v3/mosip/pms/install.sh

```diff
@@ -34,7 +34,7 @@ function installing_pms() {
   helm -n $NS install pms-policy mosip/pms-policy --set istio.corsPolicy.allowOrigins\[0\].prefix=https://$PMP_HOST --version $CHART_VERSION
 
   echo Installing pmp-ui
-  helm -n pms install pmp-ui mosip/pmp-ui  --set pmp.apiUrl=https://$INTERNAL_API_HOST/ --set istio.hosts=["$PMP_HOST"] --set imge.repository=mosipdev/pmp-ui --version 12.0.2
+  helm -n pms install pmp-ui mosip/pmp-ui  --set pmp.apiUrl=https://$INTERNAL_API_HOST/ --set istio.hosts=["$PMP_HOST"]
```

Comment:
Chart version not mentione?

Reply:
chart version is already 12.0.1-B2 which is deployed in qa-1201-b2 env sir

Reply:
I dont see the version in this command:

helm -n pms install pmp-ui mosip/pmp-ui  --set pmp.apiUrl=https://$INTERNAL_API_HOST/ --set istio.hosts=["$PMP_HOST"]